src/components/sortableList/SortableListItem.driver.new.ts

```diff
@@ -0,0 +1,38 @@
+import _ from 'lodash';
+import {SortableListItemProps} from './types';
+import {useComponentDriver, ComponentProps} from '../../testkit/new/Component.driver';
+import {useDraggableDriver} from '../../testkit/new/useDraggable.driver';
+
+export const SortableListItemDriver = (props: ComponentProps) => {
+  const driver = useDraggableDriver<SortableListItemProps>(useComponentDriver(props));
+
+  const dragUp = async (indices: number) => {
+    validateIndices(indices);
+    const data = _.times(indices, index => {
+      return {
+        translationY: -52 * (index + 1)
```

Comment:
Isn't the 52 should be a variable defined in the SortableListItem component props?

Reply:
Fixed